.github/workflows/issue-notify.yml

```diff
@@ -0,0 +1,17 @@
+name: issue-notify
+
+on:
+  issues:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    if: ${{ always() }} && github.repository == 'PDLPorters/pdl'
```

Comment:
This seems like a maintenance problem - would it make sense to have the repository pattern-matched with `/^PDLPorters/`?

Reply:
Fixed by <https://github.com/PDLPorters/devops/pull/10>.